notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
```

Comment:
Changes to this notebook exchange the old machinery for Codebook, IntensityTable, and add references to updated data on the public s3 bucket.